include/enrico/comm.h

```diff
@@ -168,6 +185,29 @@ void Comm::broadcast(std::vector<T>& values) const
   }
 }
 
+template<typename T, size_t N>
+void Comm::broadcast(xt::xtensor<T, N>& values) const
+{
+  if (this->active()) {
+    // First, make sure shape of `values` matches root's
+    auto& s = values.shape();
+    std::vector<size_t> my_shape(s.begin(), s.end());
+    std::vector<size_t> root_shape(my_shape);
+
+    this->broadcast(root_shape);
+    if (my_shape != root_shape) {
+      values.resize(root_shape);
+    }
+
+    // Next, broadcast size
+    int n = values.size();
```

Comment:
Since we don't really need to rely on this being an `int` specifically:
```suggestion
    auto n = values.size();
```